.circleci/config.yml

```diff
@@ -53,6 +53,21 @@ jobs:
       # Database setup
       - run: cd spec/dummy && bundle exec rake db:create db:schema:load
 
+      - run:
```

Comment:
@tmking, can you review this config for installing the chrome driver?

Reply:
I would recommend that you change `replace_existing:` to true to make sure that the specified version is installed.

Reply:
@tmking I'm getting `Configuration errors: In step 8 definition: step type "browser_tools/install_chrome" is not a valid type` (https://app.circleci.com/pipelines/github/network-for-good/nfg_onboarder/513/workflows/ec00a61b-ef55-4442-9d90-3b0368197d39/jobs/526) error in the setup step itself. I copied this version from the auctions circle configuration. Does that cause this setup failure, as the onboarder has a relatively old configuration file compared to auctions?

Reply:
You will need to add the browser-tools orb in order to use these commands.